src/mqtt.cpp

```diff
@@ -151,9 +151,18 @@ void sendHAConfig()
   const bool retain = true;
   const int qos = 1;
   char buff[512];
-  const int capacity = JSON_OBJECT_SIZE(10); // intentionally larger than we need.
+  const int capacity = JSON_OBJECT_SIZE(24); // intentionally larger than we need.
   StaticJsonDocument<capacity> doc;
 
+  // deviceinfo
+  const int devcapacity = JSON_OBJECT_SIZE(6);
+  StaticJsonDocument<devcapacity> deviceinfo;
+  deviceinfo.clear();
+  deviceinfo["manufacturer"] = "e-radionica";
+  deviceinfo["model"] = "Inkplate 10";
+  deviceinfo["name"] = "HomePlate";
+  deviceinfo["identifiers"][0] = "homeplate";
```

Comment:
This should be unique to the specific device in case there are multiple homeplate's on the same network. I think it should be the device's mac address so that device tracker integrations can identify it as well.

Reply:
Good point, I think this should be extended to the mqtt topics and unique_ids as well. That would make discovery less error prone. For the activity-topic the hostname could be used (ex. "homeplate/<hostname>/activity"). Any thoughts?

Reply:
I think using a unique name for the activies would be a very welcome improvment. I [noted](https://github.com/lanrat/homeplate/blob/main/src/mqtt.cpp#LL146-LL147) that it should be done a while back but never got around it it. You are welcome to tackle that if you would like. :)